src/styles.css

```diff
@@ -27,6 +27,10 @@ input[type=number]::-webkit-outer-spin-button {
   margin: 0;
 }
 
+:focus {
+    outline: 0 !important;;
```

Comment:
Doubled semicolon. Don't use `important` if it is not necessary. The common use of this keyword is fixing simple bugs in shitty css code. Try to avoid using it as much as possible because it causes lots of problems in the future.